src/Parser/GTXParser.js

```diff
@@ -0,0 +1,37 @@
+import GeoidGrid from 'Core/Geographic/GeoidGrid';
+import Extent from 'Core/Geographic/Extent';
+
+
+export default {
+    parse(gtx, options) {
+        const dataType = options.in.dataType || 'float';
+        if (dataType in ['float', 'double']) {
+            console.error(
+                '`dataType` parameter is incorrect for GTXParser.parse method. ' +
+                'This parameter must be either `double` or `float`.',
+            );
+        }
+
+        const data = new DataView(gtx);
+
+        return Promise.resolve(new GeoidGrid(
+            new Extent(
+                'EPSG:4326',
+                data.getFloat64(8),
+                // formula for the max longitude : maxLongitude = minLongitude + deltaLongitude * (nColumns - 1)
+                data.getFloat64(8) + data.getFloat64(24) * (data.getInt32(36) - 1),
+                data.getFloat64(0),
+                // formula for the max latitude : maxLatitude = minLatitude + deltaLatitude * (nRows - 1)
+                data.getFloat64(0) + data.getFloat64(16) * (data.getInt32(32) - 1),
+            ),
+            { x: data.getFloat64(24), y: data.getFloat64(16) },
+            ((latIndex, lonIndex) => {
+                if (dataType === 'float') {
+                    return data.getFloat32(40 + (data.getInt32(36) * latIndex + lonIndex) * 4);
+                } else if (dataType === 'double') {
+                    return data.getFloat64(40 + (data.getInt32(36) * latIndex + lonIndex) * 8);
+                }
```

Comment:
could use `Float64Array` in all parser?

Reply:
you could use `Float64Array.from`
You could use `Float64Array` params to split header and data?

Reply:
Unfortunately, `TypedArrays` don't allow us to manage data endianness, as they only respect hardware endianness (which is often little-endian, where data we use are big-endian). `DataView` are therefore necessary.